vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -113,25 +113,25 @@ func NewVolumeDriver(targetURL string, projectID string, hostID string, mountDir
 // map hasn't been initialized, return 1 prevent detach and remove.
 
 // Return the number of references for the given volume
-func (d *VolumeDriver) getRefCount(vol string) uint {
+func (d *VolumeDriver) getRefCount(vol string) (uint, error) {
```

Comment:
We have actually discussed this option. We can check GetInitSuccess in all the callers, but returning 1 is a bad idea as it's always ambiguous.